### PR TITLE
Add customization to For You page

### PR DESCRIPTION
**Pull request by Icarus-Xu**
add "Suggested album artists" to ForYou page and allow user to customize what they want to show on that page (Suggested albums / Suggested artists / Suggested album artists)

### CODE REVIEW

File: app/src/main/java/io/github/zyrouge/symphony/services/Settings.kt [hunk 1]
```diff
@@ -541,6 +541,10 @@ class SettingsManager(private val symphony: Symphony) {
             putString(SettingsKeys.homeTabs, tabs.joinToString(",") { it.name })
         }
         onChange.dispatch(SettingsKeys.homeTabs)
+        if (getHomeLastTab() !in tabs) {
+            setHomeLastTab(tabs.first())
+            onChange.dispatch(SettingsKeys.homeLastTab)
```
Comment (zyrouge) [inline app/src/main/java/io/github/zyrouge/symphony/services/Settings.kt:546]: Remove this line since it would be already called inside `setHomeLastTab`.

**Review by zyrouge — CHANGES_REQUESTED** [review]

**Comment by Icarus-Xu** [comment]
Removed in 561be73b6ec3ba5199b5d7aaa02a512c92865c98

File: app/src/main/java/io/github/zyrouge/symphony/services/Settings.kt (hunk 1 above)
Comment (Icarus-Xu) [reply app/src/main/java/io/github/zyrouge/symphony/services/Settings.kt]: Removed in 561be73b6ec3ba5199b5d7aaa02a512c92865c98

**Comment by zyrouge** [comment]
I'd have to rebase this once I push my local branch to here, kindly wait for that. Also, thank you for contributing.

**Review by zyrouge — APPROVED** [review]

**Comment by Icarus-Xu** [comment]
> I'd have to rebase this once I push my local branch to here, kindly wait for that. Also, thank you for contributing.

No problem, take your time.

**Pull request merged**